packages/calcite-components/src/components/navigation-user/navigation-user.scss

```diff
@@ -44,22 +43,39 @@
 
 calcite-avatar {
   @apply px-4;
+  border-radius: var(--calcite-navigation-user-avatar-corner-radius, var(--calcite-avatar-corner-radius));
+  color: var(--calcite-navigation-user-avatar-color, var(--calcite-avatar-color));
 }
 
 calcite-avatar ~ .text-container {
   @apply ps-0;
 }
 
 .full-name {
-  @apply text-0
-  ms-0
-  text-color-1
-  font-medium;
+  @apply ms-0 text-0 font-medium;
+  color: var(--calcite-navigation-user-full-name-text-color, var(--calcite-color-text-1));
 }
 
 .username {
-  @apply text-color-2;
   font-size: var(--calcite-font-size--1);
+  color: var(--calcite-navigation-user-name-text-color, var(--calcite-color-text-2));
+}
+
+:host(:hover) .button,
+:host(:focus) .button {
+  --calcite-internal-navigation-user-background-color: var(--calcite-color-foreground-2);
+}
+
+:host(:focus) .button {
+  @apply focus-inset;
+}
+
+:host(:active) .button {
+  --calcite-internal-navigation-user-background-color: var(--calcite-color-foreground-3);
+}
+
+:host([active]) .button {
+  border-block-end-color: var(--calcite-navigation-selection-border-color, var(--calcite-color-brand));
```

Comment:
Agreed.

Reply:
should this be `--calcite-navigation-accent-color` similar to `card` & `tile` ?

Reply:
Per https://github.com/Esri/calcite-design-system/wiki/tokens-component#general-naming-rules I think it should be `--calcite-navigation-accent-color` because the state change is on the :host